docs/group_faqs/gr_online_courses.rst

```diff
@@ -0,0 +1,18 @@
+Learning Material
+===================
+We're continually adding learning material to make it easier for you to adopt and use PubSub+ Cloud. 
+
+
+Udemy Courses
+--------------
+- `Fundamentals of Solace Development <https://www.udemy.com/fundamentals-of-solace-development/>`_ : The course provides an overview of the various Open Protocols and languages Solace supports,  and the concepts behind messaging with Solace. Topics covered include connecting your application to Solace, multi-protocol support, message exchange patterns, and how they relate to applications. 
+
+On Github
+------------------
+- `Samples on Github <https://dev.solace.com/get-started/send-receive-messages/>`_
```

Comment:
Suggest we say that Samples are linked under the messaging APIs and available in github and point to the GitHub org https://github.com/solacesamples

Reply:
Agreed, updated as suggested